styx-common/src/main/java/com/spotify/styx/storage/AggregateStorage.java

```diff
@@ -172,6 +172,11 @@ public void updateNextNaturalTriggerOld(WorkflowId workflowId, Instant instant)
   }
 
   @Override
+  public Map<WorkflowId, Workflow> workflows() throws IOException {
```

Comment:
Previous incarnations of this PR passed in a list of _all_ workflows to `DockerRunner.cleanup()` in order to not remove SA keys & secrets used by any workflow configuration. But that is no longer needed by this PR so could be reverted.

Might still want to implement it in a separate PR as at least the MetricsStats code seems to rely on `WorkflowCache.workflows()` returning _all_ workflows, not just the enabled and naturally triggering ones.